apps/core/priv/repo/migrations/20170909040532_create_widgets.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Core.Repo.Migrations.CreateWidgets do
+  use Ecto.Migration
+
+  def up do
+    create table(:widgets, primary_key: false) do
+      add :id, :uuid, primary_key: true
+      add :name, :string
+      add :slug, :string
+      add :description, :text, limit: 1_000
+
+      add :adaptor, :string, default: "2D"
```

Comment:
Hmm, both adaptor and adapter seem correct. Can we make this adapter pls? Only justification I can think of is that's how the Gang Of Four spell it in Adapter Pattern. Not really related but ;)